catkit/hub/tests/make_test_slabs.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import copy
+import os
+
+import ase
+import ase.build
+import ase.lattice
+import ase.io
+import ase.lattice.surface
+import ase.calculators.singlepoint
+
+path = os.path.abspath(os.path.join(os.path.dirname(__file__))) + '/unorganized/'
+
```

Comment:
I'm now writing the structures to the folder unorganized - wasn't that the intention?

Reply:
Yeah, I think the test makes more sense if it writes to the unorganized folder. I couldn't get the tests to run locally and it took me too long to figure out the correct absolute path on Travis. But great if it works like this.